influxdb3_clap_blocks/src/object_store.rs

```diff
@@ -502,6 +512,29 @@ macro_rules! object_store_config_inner {
                 )]
                 pub aws_skip_signature: bool,
 
+                /// Specify this as an alternative to `--aws-access-key-id`,
+                /// `--aws-secret-access-key`,  and `--aws-session-token`. This is a file path
+                /// argument where the format of the file is as follows:
+                ///
+                /// ```
+                /// {
+                /// "aws_access_key_id": "<key>",
+                /// "aws_secret_access_key": "<secret>",
+                /// "aws_session_token": "<token>",
+                /// "expiry": "<expiry_timestamp_ns>"
+                /// }
+                /// ```
+                ///
+                /// The server will periodically check this file path for updated contents such
+                /// that the credentials can be updated without restarting the server.
+                #[clap(
+                    id = gen_name!($prefix, "aws-credentials-file"),
+                    long = gen_name!($prefix, "aws-credentials-file"),
+                    env = gen_name!($prefix, "AWS_CREDENTIALS_FILE"),
+                    action
+                )]
+                pub aws_credentials_file: Option<String>,
```

Comment:
Should this be documented in the verbose serve help text?

Reply:
Seems like it would be useful?